src/flexmasonry.css

```diff
@@ -2,6 +2,7 @@
     display: flex;
     flex-flow: column wrap;
     align-content: space-between;
+    align-content: flex-start;
```

Comment:
Property is duplicated.

Reply:
Sure. There is a reason why `space-between` is being used here. Do you know why? I didn't want to replace it without considering why it was there. Perhaps it fixes something specific that I am not aware of ?

Reply:
No idea, really. I just saw you were duplicating it and called it so whomever else reviews and merges this can take action haha

Reply:
Sound like a good plan to me. The best thing about this fix is that it doesn't need to be merged to get the fix in production 🥇 Thanks for checking!